operator/option/config.go

```diff
@@ -256,6 +256,11 @@ const (
 	// IngressLBAnnotationPrefixes are the annotations which are needed to propagate
 	// from Ingress to the Load Balancer
 	IngressLBAnnotationPrefixes = "ingress-lb-annotation-prefixes"
+
+	// Enabled/Disables operation of kubernet-related services/controllers.
+	// Intended for operating cilium with CNI-compatible orquestrators
+	// othern than Kubernetes. (default is true)
```

Comment:
```suggestion
	// EnableK8s operation of Kubernetes-related services/controllers.
	// Intended for operating cilium with CNI-compatible orchestrators
	// other than Kubernetes. (default is true)
```

Reply:
Hi, I've pushed an updated branch with this fixes applied.